libsolidity/formal/SMTEncoder.cpp

```diff
@@ -1878,6 +1873,19 @@ Expression const* SMTEncoder::leftmostBase(IndexAccess const& _indexAccess)
 	return base;
 }
 
+Expression const* SMTEncoder::innermostTuple(TupleExpression const& _tuple)
+{
+	TupleExpression const* tuple = &_tuple;
```

Comment:
This should also handle `isInlineArray` properly - either via assertion or via breaking the while loop.

Reply:
Right!